core/api/src/app/merchants/delete-business-map-info.ts

```diff
@@ -0,0 +1,38 @@
+import { checkedToUsername } from "@/domain/accounts"
+import { MerchantsRepository } from "@/services/mongoose"
+
+export const deleteBusinessMapInfo = async ({
+  id,
+}: {
+  id: MerchantId
+}): Promise<void | ApplicationError> => {
+  const merchantsRepo = MerchantsRepository()
+
+  const result = await merchantsRepo.remove(id)
+  if (result instanceof Error) return result
+
+  return
+}
+
+export const deleteMerchantByUsername = async ({
```

Comment:
Small nit, consistent naming

```suggestion
export const deleteMerchantLocationsByUsername = async ({
```

Reply:
I'd prefer to keep it `deleteMerchantByUsername` for now because there is no dis-association as of now between a merchant and a location